docs/source/user-guide/integrations/h2o.rst

```diff
@@ -0,0 +1,171 @@
+Using deepchecks with an H2O Model
+==================================
+
+`H2O <https://docs.h2o.ai/h2o/latest-stable/h2o-docs/automl.html>`__ is
+a wildly used AutoML framework, written in Java but with multiple fully
+implemented api libraries, including one in python. In this example,
+we’ll be showing how you can effortlessly test your H2O model with
+deepchecks in the exact same way you can test any sklearn model.
+
+Building an H2O Model
+---------------------
+
+First we’ll load our dataset, using the adult dataset that can be
+downloaded from the `UCI machine learning
+repository <http://archive.ics.uci.edu/ml>`__ and is also available
+in `deepchecks.tabular.datasets`.
+
+.. code:: ipython3
+
+    import warnings
+    warnings.filterwarnings('ignore')
+    from deepchecks.tabular.datasets.classification import adult
+
+.. code:: ipython3
+
+    train_df, test_df = adult.load_data(data_format='Dataframe', as_train_test=True)
+
+Now we’ll build an H2O H2OGradientBoostingEstimator based on this
+dataset split.
+
+.. code:: ipython3
+
+    import h2o
+    h2o.no_progress()
+    h2o.init()
+
+
+H2O models can only train and predict on H2OFrame objects.
+
+.. code:: ipython3
+
+    train_h2o = h2o.H2OFrame(train_df)
+
+.. code:: ipython3
+
+    gbm = h2o.estimators.gbm.H2OGradientBoostingEstimator()
+    print('Trained!')
```

Comment:
Why the print?

Reply:
Otherwise H2O training prints a huge printout

Reply:
Alternative - add `;` at the end of previous line...